packages/react/src/components/UIShell/components/styles/_side-nav.scss

```diff
@@ -373,3 +384,15 @@ div:has(.#{$prefix}--header)
   transition: opacity $transition-expansion $standard-easing,
     background-color $transition-expansion $standard-easing;
 }
+
+// Decorative Icon
+.#{$prefix}--side-nav__decorative-icon {
+  @include focus-outline('reset');
+
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: mini-units(2) mini-units(2) 0;
+  min-block-size: mini-units(4);
```

Comment:
Let's switch these to spacing tokens, we actually shouldn't be using mini-units anywhere anymore but some of that conversion got missed over in core when we released v11.